src/material/core/ripple/ripple-ref.ts

```diff
@@ -22,7 +40,7 @@ export class RippleRef {
   state: RippleState = RippleState.HIDDEN;
 
   constructor(
-    private _renderer: RippleRenderer,
+    private _renderer: {fadeOutRipple(ref: RippleRef): void},
```

Comment:
I'd love this to be an interface, but I can't come up with a good way to solve it, other than having something like an adapter that is used here, and implemented by the `RippleRenderer`. Any ideas?

Reply:
There's nothing stopping us from having an interface inside `ripple-ref.ts`, but I didn't see a point since we don't support passing in alternate ripple renderer implementations into the ripple ref.

Reply:
yeah for sure. I guess this is just an somewhat uncommon pattern in this repo. I'm fine with it, but just wanted to see if there might be a better way.